### DISCUSSION TITLE
建议增加双指捏合缩放功能

### DISCUSSION THREAD

**Discussion by dacancc** [lollipopkit/flutter_server_box#297] [Ideas]
方便查看巨长输出及部分终端软件输出（如btop，日志等）

**Comment by dacancc** [comment]
termux缩放效果![Screenshot_20240306-220731_Termux.png](https://github.com/lollipopkit/flutter_server_box/assets/50225646/9e16b751-0597-4ec3-acec-79e6ac016eb5)

**Comment by lollipopkit** [comment]
已在1.0.807版本中实现
